netezza-plugin/docs/Netezza-batchsource.md

```diff
@@ -0,0 +1,83 @@
+# Netezza Batch Source
+
+
+Description
+-----------
+Reads from a Netezza using a configurable SQL query.
+Outputs one record for each row returned by the query.
+
+
+Use Case
+--------
+The source is used whenever you need to read from a Netezza. For example, you may want
+to create daily snapshots of a database table by using this source and writing to
+a TimePartitionedFileSet.
+
+
+Properties
+----------
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**Driver Name:** Name of the JDBC driver to use.
+
+**Host:** Host that Netezza is running on.
+
+**Port:** Port that Netezza is running on.
+
+**Database:** Netezza database name.
+
+**Import Query:** The SELECT query to use to import data from the specified table.
+You can specify an arbitrary number of columns to import, or import all columns using \*. The Query should
+contain the '$CONDITIONS' string. For example, 'SELECT * FROM table WHERE $CONDITIONS'.
+The '$CONDITIONS' string will be replaced by 'splitBy' field limits specified by the bounding query.
+The '$CONDITIONS' string is not required if numSplits is set to one. (Macro-enabled)
```

Comment:
Don't need the (Macro-enabled). I may have missed this in the other PRs, can you check if they are there in the others?

Reply:
removed